src/subgraph.template.yaml

```diff
@@ -853,6 +853,27 @@ templates:
           handler: handleSharesDecreased
         - event: RewardsWithdrawn(indexed address,uint256)
           handler: handleRewardsWithdrawn
+        - event: ClaimOnBehalfUpdated(address,bool)
+          handler: handleClaimOnBehalfUpdated
+  - kind: ethereum/contract
+    name: RewardSplitterOwnable
+    network: {{ network }}
+    source:
+      abi: RewardSplitterOwnable
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.9
+      language: wasm/assemblyscript
+      file: ./mappings/rewardSplitter.ts
+      entities:
+        - RewardSplitterOwnable
+        - RewardSplitter
+      abis:
+        - name: RewardSplitterOwnable
+          file: ./abis/RewardSplitterOwnable.json
+      eventHandlers:
+        - event: OwnershipTransferred(indexed address,indexed address)
+          handler: handleOwnershipTransferred
```

Comment:
Please keep it within RewardSplitter. You can add OwnershipTransferred to the RewardSplitter abi.